packages/fluent/scss/notification/_variables.scss

```
@@ -0,0 +1,34 @@
+@use "../core/color-system" as *;
+@use "../core/_variables.scss" as *;
+
+// Generic styles
+$info: #CEE5FF;
+$success: #DFF6DD;
+$warning: #FFF4CE;
+$error: #FDE7E9;
+
+$kendo-fluent-status-colors: (
+    "info": $info,
+    "success": $success,
+    "warning": $warning,
+    "error": $error,
+    ) !default;
+
+// Notification
+$kendo-notification-padding-x: map-get($kendo-spacing, 2) !default;
+$kendo-notification-padding-y: map-get($kendo-spacing, 1 ) !default;
+$kendo-notification-border-width: 1px !default;
+$kendo-notification-border-radius: 2px; // $kendo-border-radius-md  !default;
```

use the global variable

```suggestion
$kendo-notification-border-radius:  var( --kendo-border-radius-md )
```